src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java

```diff
@@ -162,54 +173,90 @@ public void calculateTotal() {
         return datesList;
     }
 
+    private Boolean checkIfValidHearing(DateListedType dateListedType) {
```

Comment:
Renaming the method to something like "isValidHearing" makes the SonarLint complaint goes away. Also, using boolean rather than Boolean removes the need for boxing & unboxing.

Reply:
SonarLint complaint was more with Boolean over boolean - that has been refactored